cmd/monaco/runner/runner.go

```diff
@@ -58,15 +58,21 @@ Examples:
 
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
 			log.PrepareLogging(fs, &verbose, logSpy)
-			trafficlogs.PrepareRequestResponseLogging(fs)
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			_ = cmd.Help()
 		},
+		PersistentPostRunE: func(cmd *cobra.Command, args []string) error {
+			if support.SupportArchive {
+				return support.Archive(fs)
+			}
+			return nil
+		},
 	}
 
 	// global flags
 	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "Enable debug logging")
+	rootCmd.PersistentFlags().BoolVar(&support.SupportArchive, "support-archive", false, "Create support archive")
```

Comment:
A bit more documentation on what a support archive will contain and where it can be found would be helpful for a user just seeing the flag

Reply:
yeah totally forgot about the actual feature :D will do